bindings/pyroot/pythonizations/test/rdfdescription.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import ROOT
+
+class RDFDescriptionTest(unittest.TestCase):
+    """
+    Testing of RDFDescription pythonization
+    """
+    def test_repr(self):
+        """
+        Test supported __repr__
+        """
+        df1 = ROOT.RDataFrame(1);
+        ref1 = ("Empty dataframe filling 1 row\n"
```

Comment:
Assuming that there are tests in RDataFrame for `AsString` that check it does the right thing, this test could be simply `         self.assertEqual(df1.Describe().AsString(), repr(df1.Describe()));`, i.e. no need to test again what is tested in the `AsString` tests.

Reply:
I am applying this change and creating a new commit now. Everything else I keep unchanged (so far).